src/ProxyManager/Factory/RemoteObject/AdapterInterface.php

```diff
@@ -13,8 +13,6 @@ interface AdapterInterface
      * Call remote object
      *
      * @param array<int, mixed> $params
-     *
-     * @return mixed
      */
-    public function call(string $wrappedClass, string $method, array $params = []);
+    public function call(string $wrappedClass, string $method, array $params = []): mixed;
```

Comment:
This change is a BC break, because a child class declaring no return type is not considered compatible according to current PHP 8.x variance semantics